### DISCUSSION TITLE
Bug Fixes and initial libretro core

### DISCUSSION THREAD

**Discussion by jtothebell** [jtothebell/fake-08#175] [General]
All Platforms
-Fix behavior when drawing coordinates beyond sprite sheet bounds
-Fix mget bounds check being off by one on upper bounds (fixes some visual artifacts in map() and tline() calls)
-Fix some text wrapping bugs
-Fix some text coloring bugs
-Fix bugs keeping track of smaller than default character size and line heights
-Fix a bug with music playback where looping sections were cut short (thanks @fringd)

Miyoo Mini (standalone app)
-Add double buffering to fix occasional screen tearing

New Platforms
-Add experimental libretro builds for Miyoo Mini and Android

<hr /><em>This discussion was created from the release <a href='https://github.com/jtothebell/fake-08/releases/tag/v0.0.2.19'>Bug Fixes and initial libretro core</a>.</em>

**Comment by deliyuerek85** [comment]
Thank you very much for the retroarch/libretro core. It works like a charm on my android system.

**Comment by 54MU3L3990** [comment]
OPK isn't working on rg350. If you try to boot a pico game, It goes black for a split second and sends you back to the menu.
